### DISCUSSION TITLE
Text to speech component

### DISCUSSION THREAD

**Discussion by Bebrich** [FakeFishGames/Barotrauma#11450] [Ideas and suggestions]
Not sure if this would suit a mod better, but it would be cool if there was some thing that could say text out loud. Not much that could go wrong with that.

**Comment by mygamingaccount** [comment]
aeiouaeiouaeiouaeiouaeiouaeiouaeiouaeiouaeiouaeiouaeiouaeiouaeiouaeiouaeiouaeiouaeiouaeiouaeiouaeiouaeiouaeiouaeiouaeiouaeiouaeiouaeiouaeiouaeiouaeiouaeiouaeiouaeiouaeiouaeiouaeiouaeiouaeiouaeiouaeiouaeiouaeiouaeiouaeiouaeiouaeiouaeiouaeiouaeiouaeiouaeiouaeiouaeiouaeiouaeiouaeiouaeiouaeiouaeiouaeiouaeiouaeiouv

Devs give t2s comp pls
